install/kubernetes/helm/istio/charts/kiali/templates/serviceaccount.yaml

```diff
@@ -8,10 +8,10 @@ imagePullSecrets:
 {{- end }}
 {{- end }}
 metadata:
-  name: kiali-service-account
+  name: kiali
   namespace: {{ .Release.Namespace }}
   labels:
-    app: kiali
+    app: kiali-service-account
```

Comment:
i think they changed the wrong "kiali" :)
I think the app label remains "kiali" - its the metadata.name that has to change (see line 11 above)

Reply:
fixed in 863797f